client/state/reader/reducer.js

```diff
@@ -11,11 +11,13 @@ import feeds from './feeds/reducer';
 import sites from './sites/reducer';
 import start from './start/reducer';
 import posts from './posts/reducer';
+import related_posts from './related-posts/reducer';
```

Comment:
hmmm i thought we were keeping it in line with the api responses. i guess not. will fix.